### ISSUE TITLE
Preview suggested changes for insufficient contrast ratio errors

### ISSUE THREAD

**Issue by jgardella** [Khan/tota11y#21]
I think it would be useful to be able to preview the suggested new colors to fix insufficient contrast ratio errors; perhaps a toggle button next to the error that toggles between the original colors and the suggested colors.

**Comment by jdan** [comment]
:+1: Awesome suggestion

**Comment by jgardella** [comment]
This requires being able to include functionality in an error description. Is there a "correct" way to do this with the current plugin system?

**Comment by jdan** [comment]
Nope :) May need to settle #28 first so you can click items in the description, though.

**Comment by jdan** [comment]
@jgardella Oh I see you're way ahead of me. I'll review your PR later this afternoon. Just make sure you've [signed the CLA](http://khanacademy.org/r/cla) if you haven't already as per [the instructions in CONTRIBUTING.md](https://github.com/Khan/tota11y/blob/master/CONTRIBUTING.md)
